libraries/chain/event_evaluator.cpp

```diff
@@ -32,10 +33,38 @@
 
 namespace graphene { namespace chain {
 
+namespace 
+{
+   /// searches for the manager in event => event_group => sport
+   bool is_manager( const database& db, const event_id_type& event_id, const account_id_type& manager_id )
+   {
+      const event_object& event_obj = event_id(db);
+      if( event_obj.manager == manager_id ) 
+         return true;
+
+      const event_group_object& event_group_obj = event_obj.event_group_id(db);
+      if( event_group_obj.manager == manager_id )
+         return true;
+
+      const sport_object& sport_obj = event_group_obj.sport_id(db);
+      return sport_obj.manager == manager_id;
```

Comment:
Need one additional test to allow `GRAPHENE_WITNESS` to still do anything they want (multisig)